### ISSUE TITLE
File Explorer app update on Firestick  issue again

### ISSUE THREAD

**Issue by DB628** [visnkmr/file_explorer#250]
Hello, your file Explorer says there is an update for the fire-stick, but no matter what we try it doesn't update, I've tried rebooting the FS, I've tried uninstalling and still says it has an update. This has happened before and then a month later it finally update, please advise on how to resolve this issue, Thanks again

DJB

**Comment by DB628** [comment]
I have tried downloading the app from your GitHub and then installing it, and it keeps saying app not installed. Not sure what else to do to side-load the app since you're having issues with the Amazon app store

**Comment by CodingPassionYT** [comment]
Same here says update available but i have latest version tag from this github.

**Comment by Simi2437** [comment]
Same here

**Comment by visnkmr** [comment]
even after uninstalling the amazon appstore app and then trying to installing the apk from our appstore it fails, seems an issue with the apk then, let me try to replicate the issue.

**Comment by Simi2437** [comment]
It has also the version string of the github apk. Sonit is the same Version.

**Comment by DB628** [comment]
> even after uninstalling the amazon appstore app and then trying to installing the apk from our appstore it fails, seems an issue with the apk then, let me try to replicate the issue.

At first when I side-loaded the APK, I thought the issue might of been an issue with the OS of FS, as I had tried it on one of the FS with OS 6, but then I tried it on a FS Max OS 7 and the same issue. Just in-case you were wondering. 

DJB

**Comment by DB628** [comment]
> even after uninstalling the amazon appstore app and then trying to installing the apk from our appstore it fails, seems an issue with the apk then, let me try to replicate the issue.

Were you able to look into this ??

**Comment by visnkmr** [comment]
Yes actually it was an issue from my end, ended up uploading an older version of the app, sorry about that . Will upload latest apk now. Once again thanks for reminding me.

**Comment by visnkmr** [comment]
> > even after uninstalling the amazon appstore app and then trying to installing the apk from our appstore it fails, seems an issue with the apk then, let me try to replicate the issue.
> 
> Were you able to look into this ??

Available on our appstore and @ https://github.com/visnkmr/file_explorer/releases/tag/20, starting now.

submitted to amazon appstore, will be available in 1-2 days.

**Comment by DB628** [comment]
Still could not sideload on OS 6, see screen shot, I even tried renaming it
still get the same message see image below, thanks for looking into this it
is greatly appreciated

Does the new update only work on OS 7 ❓

**Comment by DB628** [comment]
Update

As mentioned above, was still not able to install and update, So I decided to uninstall and install the new APK from a fresh install. and all is good and working now. I will not side load on my other devices, to see if Amazon updates their system, Ill keep you posted if they do.

Thanks again for the support 

DJB
